doc/ADMIN.md

```diff
@@ -17,61 +23,71 @@ You must also add your public key to your Gitea profile.
+By default, NGINX is configured with a maximum value for uploading files at 200 MB. It's possible to change this value on `/etc/nginx/conf.d/__DOMAIN__.d/__APP__.conf`.
 
-`yunohost app upgrade gitea`
+```nginx
+client_max_body_size 200M;
+```
 
-### Backup
+Don't forget to restart Gitea:
 
-This application now uses the core-only feature of the backup. To keep the integrity of the data and to have a better guarantee of the restoration it is recommended to proceed as follows:
+```bash
+sudo systemctl restart __APP__.service`.
+```
 
-- Stop Gitea service with this command:
+> These settings are restored to the default configuration when updating Gitea. Remember to restore your configuration after all updates.
 
-`systemctl stop gitea.service`
+## Upgrade
 
-- Launch Gitea backup with this command:
+From command line:
 
-`yunohost backup create --app gitea`
+```bash
+yunohost app upgrade __APP__
+```
 
-- Backup your data with your specific strategy (could be with rsync, borg backup or just cp). The data is generally stored in `/home/yunohost.app/gitea`.
-- Restart Gitea service with theses command:
+If you want to bypass the safety backup before upgrading, run:
 
-`systemctl start gitea.service`
+```bash
+yunohost app upgrade --no-safety-backup __APP__
+```
 
-### Remove
+## Backup
 
-Due of the backup core only feature the data directory in `/home/yunohost.app/gitea` **is not removed**. It must be manually deleted to purge user data from the app.
+This application now uses the core-only feature of the backup. To keep the integrity of the data and to have a better guarantee of the restoration it is recommended to proceed as follows:
 
-### LFS setup
-To use a repository with an `LFS` setup, you need to activate it on `/opt/gitea/custom/conf/app.ini`
+- Stop Gitea service:
 
-```ini
-[server]
-LFS_START_SERVER = true
-LFS_HTTP_AUTH_EXPIRY = 20m
-```
-By default, NGINX is configured with a maximum value for uploading files at 200 MB. It's possible to change this value on `/etc/nginx/conf.d/my.domain.tld.d/gitea.conf`.
+```bash
+systemctl stop __APP__.service
 ```
-client_max_body_size 200M;
+
+- Launch Gitea backup:
+
+```bash
+yunohost backup create --app __APP__
 ```
-Don't forget to restart Gitea `sudo systemctl restart gitea.service`.
 
-> These settings are restored to the default configuration when updating Gitea. Remember to restore your configuration after all updates.
+- Backup your data with your specific strategy (could be with rsync, borg backup or just cp). The data is generally stored in `/home/yunohost.app/__APP__`.
+- Restart Gitea service:
+
+```bash
+systemctl stop __APP__.service
+```
 
-### Git command access with HTTPS
+## Remove
 
-If you want to use the Git command (like `git clone`, `git pull`, `git push`), you need to set this app as **public**.
+Due of the backup core only feature the data directory in `/home/yunohost.app/__APP__` **is not removed**. It must be manually deleted to purge user data from the app.
```

Comment:
Why not, but that's the same stuff for every app with a data dir ...

Anyway, nowadays `yunohost app remove` has a `--purge` option, and i think with packaging v2 there may be a modal asking admins wether or not to purge data ?